lib/Installation/Popups/YesNoPopup.pm

```diff
@@ -22,7 +22,9 @@ sub new {
 sub init {
     my $self = shift;
     $self->{btn_yes} = $self->{app}->button({id => 'yes'});
-    $self->{btn_no} = $self->{app}->button({id => 'no'});
+    # The id of the "No" button varies depending on the YaST version.
+    # Therefore we use a regex
+    $self->{btn_no} = $self->{app}->button({id => qr/no|no_button/});
```

Comment:
Nice one!

Reply:
> Nice one!

Agree:)

Reply:
@rakoenig Does your VR already includes this code? Just wonder if it works correctly.

Reply:
Yes, the code ist in the VR. The late changes were only affecting 3 YAML files that hat a conflict when I was rebasing to the current master.

Reply:
I would prefer to use `label` here, which is valid for both products and doesn't get the whole json with a regex (regex should be used when there is not other way), It would not need a comment like that (I really prefer not read comments when there is not need and for us is obvious that products are different) and also I think this would match any string after `no*`, so if the screen would include some `no_whatever` will also match it, unlikely but confusing, when label would be really clear. wdyt?

Reply:
I stick with the regex.